src/styles/pages/store/cart/index.module.scss

```
@@ -0,0 +1,176 @@
+        'items'
+        'summary';
+      grid-template-columns: 1fr;
+    }
+
+    .storeButton {
+      background-color: var(--theme-primary-2);
+      border-radius: 0.625rem;
+      color: vars.$light-background;
+      min-height: 2rem;
+      padding: 0 0.5rem;
+      width: 100%;
+    }
+
+    .title {
+      grid-area: title;
+      margin-bottom: 2rem;
+    }
+
+    .placeOrder {
+      background: var(--primaries-primary-2, #62B0FF);
+      border: 1px solid var(--theme-accent-line-2, #E4E4E4);
+      border-radius: 10px;
+      color: vars.$light-background;
+      font-size: 20px;
+      font-weight: 700;
+      grid-area: placeOrder;
+      height: 2.5rem;
+      margin-bottom: 2rem;
+
+      &:disabled {
+        background-color: transparent;
+        color: vars.$disabled;
+        cursor: not-allowed;
+      }
+
+      .confirming {
+        background: vars.$success-1;
+      }
+    }
+
+    .cartCard {
+      border-radius: 0.625rem;
+      box-shadow: 0 0.25rem 0.25rem var(--theme-shadow);
+
+      .header {
+        align-items: center;
+        background-color: var(--theme-surface-1);
+        border-top-left-radius: 0.625rem;
+        border-top-right-radius: 0.625rem;
+        display: flex;
+        height: 3.75rem;
+        padding: 1.5rem 1rem;
+      }
+    }
+
+    .confirmation {
+      align-items: center;
+      background-color: var(--theme-surface-1);
+      display: flex;
+      flex-direction: column;
+      gap: 1rem;
+      grid-area: confirmation;
+      margin-bottom: 2rem;
+      padding: 2rem;
+      text-align: center;
+
+      div {
+        display: flex;
+        flex-wrap: wrap;
+        gap: 1rem;
+        justify-content: center;
+        width: 100%;
+
+        a {
+          flex-grow: 1;
+          max-width: 11.625rem;
+        }
+      }
+    }
+
+    .items {
+      display: flex;
+      flex-direction: column;
+      grid-area: items;
+      margin-bottom: 2rem;
+      padding-bottom: 1rem;
+
+      .emptyCart {
+        align-items: center;
+        display: flex;
+        flex-direction: column;
+        gap: 0.5rem;
+        padding: 2rem 0;
+        text-align: center;
+        width: 100%;
+
+        a {
+          margin: 0 1rem;
+          width: 12.5rem;
+        }
+      }
+    }
+
+    .pointsCard {
+      grid-area: summary;
+      margin-bottom: 2rem;
+
+      .points {
+        display: flex;
+        flex-direction: column;
+        gap: 0.5rem;
+        justify-content: space-around;
+        padding: 0.5rem 0.625rem;
+  
+        > div {
+          align-items: baseline;
+          display: flex;
+          justify-content: space-between;
+  
+          span {
+            text-align: end;
+          }
+        }
+  
+        hr {
+          border: 0.0625rem solid var(--theme-accent-line-1-transparent);
+        }
+      }
+    }
+
+    .eventPicker {
+      grid-area: event;
+      margin-bottom: 0.5rem;
+
+      @media (max-width: vars.$breakpoint-md) {
```

```suggestion
      @media screen and (width < vars.$breakpoint-md) {
```

I want to try using a consistent syntax for all of the media queries in the repo